src/rattler_build_conda_compat/jinja/objects.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+
+class _StubEnv:
+    """A class to represent the env object used in rattler-build recipe."""
+
+    def get(self, env_var: str, default: str | None = None) -> str:
+        if default:
+            return f"""${{{{ env.get("{env_var}", default="{default}") }}}}"""
+
+        return f"""${{{{ env.get("{env_var}")}}}}"""
+
+    def exists(self, env_var: str) -> str:
+        return f"""${{{{ env.exists("{env_var}") }}}}"""
+
+
+def _stub_compatible_pin(*args, **kwargs) -> str:  # noqa: ARG001, ANN003, ANN002
+    return f"${{{{ compatible_pin {args[0]} }}}}"
+
+
+def _stub_subpackage_pin(*args, **kwargs) -> str:  # noqa: ARG001, ANN003, ANN002
+    return f"subpackage_pin {args[0]}"
```

Comment:
I am not sure I understand the difference. Here you return simple string, but in other functions you return `${{ .... }}`.

Reply:
made them all to return just the simple string